src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/UserManagement/UserManagementController.cs

```diff
@@ -0,0 +1,142 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using GovUk.Education.ExploreEducationStatistics.Admin.Services.Interfaces;
+using GovUk.Education.ExploreEducationStatistics.Admin.ViewModels;
+using GovUk.Education.ExploreEducationStatistics.Common.Extensions;
+using GovUk.Education.ExploreEducationStatistics.Common.Model;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Controllers.Api.UserManagement
+{
+    [ApiController]
+    [Route("api/user-management")]
+    [Authorize(Policy = "CanManageUsersOnSystem")]
+    public class UserManagementController : ControllerBase
+    {
+        private readonly IUserManagementService _userManagementService;
+        private readonly IUserRoleService _userRoleService;
+
+        public UserManagementController(IUserManagementService userManagementService,
+            IUserRoleService userRoleService)
+        {
+            _userManagementService = userManagementService;
+            _userRoleService = userRoleService;
+        }
+
+        [HttpGet("users")]
+        [ProducesResponseType(200)]
+        public async Task<ActionResult<List<UserViewModel>>> GetUserList()
+        {
+            return await _userManagementService
+                .ListAllUsers()
+                .HandleFailuresOrOk();
+        }
+
+        [HttpGet("users/{id:guid}")]
+        [ProducesResponseType(200)]
+        public async Task<ActionResult<UserViewModel>> GetUser(Guid id)
+        {
+            return await _userManagementService
+                .GetUser(id)
+                .HandleFailuresOrOk();
+        }
+
+        [HttpPut("users/{userId}")]
```

Comment:
I've made this change. The param type was string but I've changed it to Guid which I think is better, even though the service only uses String. The reason for this is because it only uses ApplicationUser rather than User.